Dockerfile/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM osrf/ros:galactic-desktop
+FROM osrf/ros:rolling-desktop
```

Comment:
Just a heads up that Rolling will move to Jammy very soon, but Ignition debs will take a bit longer to come along. So this may be broken for a while.

Reply:
An alternative would be to start from a bare Focal image and install ROS and Ignition there. It's slower, but it should work until we have Ignition Jammy debs.